### PR TITLE
create model

### PR DESCRIPTION
**Pull request by thucpv-1748**
redmine: https://edu-redmine.sun-asterisk.vn/issues/8883

### CODE REVIEW

File: app/Model/Category.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    protected $fillable = [];
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+
+    public function getFilm()
+    {
+        return $this->hasMany('App\Model\Film');
+    }
```
Comment (framgiaciv3) [inline app/Model/Category.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Model/Category.php</br>**Line**: 28</br>The closing brace for the class must go on the next line after the body

File: app/Model/Coupon.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    /**
+     * @var string
+     */
+    protected $table = 'coupons';
+
+    /**
+     * @var array
+     */
+    protected $fillable = [];
```
Comment (framgiaciv3) [inline app/Model/Coupon.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Model/Coupon.php</br>**Line**: 20</br>The closing brace for the class must go on the next line after the body

File: app/Model/Film.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        return $this->belongsTo('App\Model\Category', 'category_id');
+    }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function timeShow()
+    {
+        return $this->hasMany('App\Model\TimeShow');
+    }
```
Comment (framgiaciv3) [inline app/Model/Film.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Model/Film.php</br>**Line**: 43</br>The closing brace for the class must go on the next line after the body

File: app/Model/Room.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        return $this->belongsTo('App\Model\Store', 'store_id');
+    }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function seat()
+    {
+        return $this->hasMany('App\Model\Seat');
+    }
```
Comment (framgiaciv3) [inline app/Model/Room.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Model/Room.php</br>**Line**: 41</br>The closing brace for the class must go on the next line after the body

File: app/Model/FastFood.php [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+    /**
+     * @var string
+     */
+    protected $table = 'fast_foods';
+
+    /**
+     * @var array
+     */
+    protected $fillable = [];
```
Comment (framgiaciv3) [inline app/Model/FastFood.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Model/FastFood.php</br>**Line**: 25</br>The closing brace for the class must go on the next line after the body

File: app/User.php [hunk 6]
```diff
@@ -16,7 +16,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password',
+        'email', 'password',
```
Comment (framgiaciv3) [inline app/User.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ValueNoNewline</br>**Severity**: error</br>**File**: app/User.php</br>**Line**: 19</br>Each value in a multi-line array must be on a new line

File: app/User.php [hunk 7]
```diff
@@ -29,11 +29,42 @@ class User extends Authenticatable
```
Comment (framgiaciv3) [inline app/User.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ValueNoNewline</br>**Severity**: error</br>**File**: app/User.php</br>**Line**: 28</br>Each value in a multi-line array must be on a new line

File: app/User.php [hunk 8]
```diff
@@ -29,11 +29,42 @@ class User extends Authenticatable
...
+    {
+        return $this->belongsToMany(Role::class);
+    }
+
+    /**
+     * @param string|array $roles
+     */
+    public function authorizeRoles($roles)
+    {
+        if (is_array($roles)) {
```
Comment (framgiaciv3) [inline app/User.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/User.php</br>**Line**: 45</br>Blank line found at start of control structure

File: app/User.php [hunk 9]
```diff
@@ -25,15 +26,46 @@ class User extends Authenticatable
...
+            return $this->hasAnyRole($roles) ||
+                abort(401, 'This action is unauthorized.');
```
Comment (BunnyPi04) [inline app/User.php]: i18n

File: app/Model/Store.php [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+namespace App\Model;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Store extends Model
+{
+    /**
+     * @var string
+     */
+    protected $table = 'stores';
```
Comment (BunnyPi04) [inline app/Model/Store.php]: tên bảng đặt đúng convention thì sẽ không cần tới biến $table này nữa nhé

**Pull request merged**